README.md

```diff
@@ -1,20 +1,101 @@
-# Liverpool Telescope facility module for the TOM Toolkit
+# Liverpool Telescope facility module for the TOM Toolkit v0.3.0
```

Comment:
I think that some form of this documentation should also live in the TOM Toolkit docs. Putting a small blurb [here](https://github.com/TOMToolkit/tom_base/blob/master/docs/api/tom_observations/facilities.rst) would help raise awareness about the module.